crates/fj-app/src/main.rs

```diff
@@ -66,3 +67,11 @@ fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+fn no_model_error() -> anyhow::Error {
+    anyhow!(
+        "You must specify a model to start Fornjot.\n\
+        - Pass a model as a command-line argument. See `fj-app --help`.\n\
+        - Specify a default model in the configuration file."
+    )
```

Comment:
This error message isn't quite true anymore. It only appears in the context of exporting.

I think it would be nicer to update it, to specifically say you need a model *when exporting*. But this is optional, and I'd happily merge the pull request with the error message staying like this.